RHOAI-10948 monitoring book restructure (#394)
opendatahub-io/opendatahub-documentation: modules/installing-trustyai-service.adoc

:_module-type: CONCEPT

[id='installing-trustyai-service_{context}']
= Installing the TrustyAI service for a data science project

[role='_abstract']

> Install TrustyAI into your project and provides access to its features for all models deployed within that project so that data scientists can monitor models. An instance of the TrustyAI service is required for each data science project, or namespace, that contains models that your team wants to monitor. 

Reviewer: ```suggestion
Install TrustyAI into your project and provide access to its features for all models deployed within that project so that data scientists can monitor models. An instance of the TrustyAI service is required for each data science project, or namespace, that contains models that your team wants to monitor. 
```
Minor typo - might have been my suggestion!

Author: I think I can delete that comma after namespace too?

> For each data science project (namespace) that contains models for which the data scientists on your team want to monitor, you need to install an instance of the TrustyAI service. This action installs TrustyAI into your project and provides access to its features for all models deployed within that project.

Reviewer: ```suggestion
Install TrustyAI into your project and provides access to its features for all models deployed within that project so that data scientists can monitor models for bias. An instance of the TrustyAI service is required for each data science project, or namespace, that contains models that your team wants to monitor. 
```
Rearranged slightly - felt like the second sentence gave a larger overview of this topic than the first. Also modified to be less feature focused but feel free to change as you see fit if I've mucked it up 😄

ifdef::upstream[]

> You can install an instance of the TrustyAI service by using the {productname-short} dashboard or by using the OpenShift command-line interface (CLI).

Reviewer: ```suggestion
Use the {productname-short} dashboard or the OpenShift command-line interface (CLI) to install an instance of the TrustyAI service.
```

endif::[]

[NOTE]
====

> You should install only one instance of the TrustyAI service in a project. The presence of multiple instances in the same project can result in unexpected behavior.

Reviewer: ```suggestion
Install only one instance of the TrustyAI service in a project. Multiple instances in the same project can result in unexpected behavior.
```
I kind of feel like the second sentence is implied by the first, so I also suggest:
```suggestion
Install only one instance of the TrustyAI service in a project.
```
